apph-front/.eslintrc

```
@@ -9,6 +9,7 @@
     "prettier"
   ],
   "rules": {
-    "prettier/prettier": 2
+    "prettier/prettier": 2,
+    "linebreak-style": ["error", "unix"]
```

```suggestion
    "prettier/prettier": [
      "error",
      {
        "endOfLine": "auto"
      }
    ]
```

Avec cette config ça marche chez moi sans rien changer dans l'IDE